command.go

```diff
@@ -248,6 +249,20 @@ func ExecuteCopy(c parser.Command, _ *VHS) {
 	_ = clipboard.WriteAll(c.Args)
 }
 
+// ExecuteEnv sets env with given key-value pair.
+func ExecuteEnv(c parser.Command, v *VHS) {
+	_ = os.Setenv(c.Options, c.Args)
```

Comment:
Need to also un-set these environment variables after they are used(?)

Reply:
Does this affect the actual users session? I imagine it doesn't and will disappear after the recording is over. If it does persist after the session is done then we could first save the user environment and then set the environment to the `oldEnvironment` afterwards.

Reply:
Ok it doesn't affect and disappears.